guest-tools/image/create-td-image.sh

```diff
@@ -79,6 +79,18 @@ check_tool() {
     [[ "$(command -v $1)" ]] || { error "$1 is not installed" 1>&2 ; }
 }
 
+# On Ubuntu noble 24.04
+# if passt is install and virt-customize runs as root, virt-customize
+# will fail, this is a work-around for this issue
+# Furthermore, we see some instability to reach ubuntu archive when
+# passt is used on 24.10, so for now, we decide to remove it
+# for both 24.04 and 24.10
+workaround_passt() {
+    if command -v passt 2>&1 > /dev/null ; then
+       apt autoremove passt -y
```

Comment:
I think we should ask the user to confirm here before removing a package from their system

Reply:
I was hesitant but i agree since you noticed it.

Reply:
Done : 8c171bb8a4dc6ea4d34c0d865c5e1a28e5fa31f9